zap/src/main/java/org/zaproxy/zap/db/paros/ParosTableAlertTag.java

```diff
@@ -201,11 +201,16 @@ public synchronized Map<String, String> getAllTags() throws DatabaseException {
     @Override
     public synchronized List<RecordAlertTag> getAllRecords() throws DatabaseException {
         try {
+
             List<RecordAlertTag> result = new ArrayList<>();
-            try (ResultSet rs = psGetAllTags.executeQuery()) {
-                RecordAlertTag rat;
-                while ((rat = build(rs)) != null) {
-                    result.add(rat);
+            try (PreparedStatement psReadScan =
+                    getConnection()
+                            .prepareStatement("SELECT * FROM alert_tag ORDER BY alert_id ASC")) {
```

Comment:
The initial key tag "alerttag.ps.getalltags" did not contain all columns. This would cause the initial code base to fail with a sqlexception tag_id doesn't exist. 
I added that key tag "alerttag.ps.getalltags2" to include all the columns.

Reply:
You can't use that there. One is for hsqldb and the other is based on paros db. 
I had to update both for it to work if someone uses hsqldb.

Reply:
I wasn't questioning the need for different SQL. I was just wondering why it's literal here instead of getting what was added to hsqldb.properties

Reply:
Oh okay, got ya.